lib/resources/payments.js

```diff
@@ -43,7 +43,7 @@ module.exports = function (api) {
       }, callback)
     },
 
-    capture(paymentId, amount, callback) {
+    capture(paymentId, amount, currency = 'INR', callback) {
```

Comment:
@rvnew 

This will cause backward compatibility issue,
Please write the function in such a way that if the third argument is a function , consider it as callback , else check if its a string and try to use it as currency , which would server your case and also ensures Backward compatibility.

Reply:
We can change the function signature in next version bump.